cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,15 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+                Example: `# Create a Cluster Workflow Template:
+argo cluster-template create FILE1
```

Comment:
```suggestion
        Example: `# Create a Cluster Workflow Template:
  argo cluster-template create FILE1
```

- first line is [failing](https://github.com/argoproj/argo-workflows/actions/runs/6686280350/job/18165470480?pr=12101) `lint` (`gofmt`)
- second line is failing `codegen` and should be indented by 2 spaces by convention